SamsungHealth-iOS/SamsungHealth-iOS/Scene(yi)/Weight/YinWeightViewController.swift

```diff
@@ -8,22 +8,107 @@
 import UIKit
 
 class YinWeightViewController: UIViewController {
-
+    
+    // MARK: - Properties
+    
+    var weight = Float(0.0)
+    
+    // MARK: - IBOutlet
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var dateLabel: UILabel!
+    
+    @IBOutlet var msgLabels: [UILabel]!
+    @IBOutlet var titleLabels: [UILabel]!
+    @IBOutlet var typeLabels: [UILabel]!
+    
+    @IBOutlet weak var weightPickerView: UIPickerView!
+    @IBOutlet weak var weightTextField: UITextField!
+    @IBOutlet weak var muscleTextField: UITextField!
+    
+    // MARK: - VC Life Cycle (or Cell Life Cycle)
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        setUI()
     }
     
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+    override func viewWillLayoutSubviews() {
+        initPickerView()
+    }
+    
+    // MARK: - Function
+    
+    private func setUI() {
+        setFont()
+        setPickerView()
+        setTextField()
+    }
+    
+    private func setFont() {
+        titleLabel.font = .SshFontH2
+        dateLabel.font = .SshFontB1
+        msgLabels.forEach { $0.font = .SshFontB3 }
+        titleLabels.forEach { $0.font = .NotoSans(.medium, size: 15) }
+        typeLabels.forEach { $0.font = .NotoSans(.medium, size: 13)}
+    }
+    
+    private func initPickerView() {
+        weightPickerView.subviews[1].backgroundColor = .clear
     }
-    */
+    
+    private func setPickerView() {
+        weightPickerView.delegate = self
+        weightPickerView.dataSource = self
+    }
+    
+    private func setTextField() {
```

Comment:
얘도 UI 커스텀 관련된 부분이면

```suggestion
    private func setTextFieldUI() {
```

이름을 이런식으로,, 하는 편
왜냐면 TextFieldDelegate를 쓰면 또
private func setTextField() { textField.delegate = self }
머 이런 애 넣어줄 상황이 생길수도 있으니까..
(앱에 의미없는 텍스트필드가 별로 없어서 텍스트필드 있는 뷰에선 항상 이랬던듯?)



그래서 결론적으론

뷰디드로드(){ 셋유아이() 셋딜리게이트() }


private func setUI() {
    setTextFieldUI()
    setLabelUI()
..dnddㅇ웅얼웅얼..
}


private func setDelegate() {
    setTextField()
    setCollectionView()
}


모 이렇게 한다던가....
기능에 따라 나눈다는 건데 그때그때 유동적으로 하면 댈듯

Reply:
헉 선배 완전 i got a boy 
Thank you so Much 임
진짜 jinshim으로 💋